explore/beta/sasfit_compare.py

```diff
@@ -375,7 +374,7 @@ def data_file(name):
 
 def load_sasfit(path):
     data = np.loadtxt(path, dtype=str, delimiter=';').T
-    data = np.vstack((map(float, v) for v in data[0:2]))
+    data = np.vstack(map(float, v) for v in data[0:2])
```

Comment:
This should be failing. For version 2.2.6 that I happen to have installed, `np.vstack(generator)` gives
`TypeError: arrays to stack must be passed as a "sequence" type such as list or tuple.`

This is a bug in the original, which should have used [...] rather than (...) when forming the sequence to send to vstack, so not an error introduced by this PR.